### DISCUSSION TITLE
指南纠错-“客户端-单页应用支持‘中”update“小节中的代码有误

### DISCUSSION THREAD

**Discussion by KevinYuHere** [walinejs/waline#471] [Q&A] [answered]
原代码：
// 在 `/` 路由
const waline = Walien({                                  // 这里n、e两个字母写反了
  serverURL: 'https://example.com',
});

/* 用户: 一段时间后，路由变动到 `/a.html` */

waline.update(); // 现在 waline 将开始清除评论并显示加载状态
// 一段时间后，计数器和评论将全部更新

waline.update({
  lang: 'en',
  login: 'disable',
}); // 这将使 Waline 以英文显示，并禁用用户登录
正确代码：
// 在 `/` 路由
const waline = Waline({                                  // 这里可以这样写
  serverURL: 'https://example.com',
});

/* 用户: 一段时间后，路由变动到 `/a.html` */

waline.update(); // 现在 waline 将开始清除评论并显示加载状态
// 一段时间后，计数器和评论将全部更新

waline.update({
  lang: 'en',
  login: 'disable',
}); // 这将使 Waline 以英文显示，并禁用用户登录

**Comment by KevinYuHere** [comment]
另外，字数的统计似乎有问题，只能统计汉字，不能统计英文字符和数字？

**Reply by Mister-Hope** [reply]
我们做的是“字词”的功能，中文“我需要苹果”是5个字，英文的 `I need an apple` 是四个词。国外很少用总长度这个概念。

**Reply by KevinYuHere** [reply]
好的，是我理解有误

**Comment by Mister-Hope** [comment, marked as answer]
我反复的对照了两分钟，你的这一大段没有排版的文字，是不是就想说有一处 `Waline` 拼错了？我没看到其他实质性的不同

**Reply by Mister-Hope** [reply]
这个问题已经修复了，下次方便的话直接提一个PR就好。

**Reply by KevinYuHere** [reply]
OK
